frontend/public/components/deployment-config.tsx

```diff
@@ -33,6 +32,8 @@ import {
   WorkloadTableRow,
   WorkloadTableHeader,
 } from './workload-table';
+import PodRingDataContoller from './../../packages/dev-console/src/components/topology/PodRingDataController';
+import PodsRing from './../../packages/dev-console/src/components/PodsRing';
```

Comment:
Bad dependencies. Do not depend on dev-console from public folder. Move common code to `@console/shared`